CRM/Report/Form/Case/Demographics.php

```diff
@@ -371,10 +385,6 @@ public function groupBy() {
     $this->_groupBy = CRM_Contact_BAO_Query::getGroupByFromSelectColumns($this->_selectClauses, $groupBy);
   }
 
-  public function orderBy() {
-    $this->_orderBy = "ORDER BY {$this->_aliases['civicrm_contact']}.sort_name, {$this->_aliases['civicrm_contact']}.id, {$this->_aliases['civicrm_case']}.id";
-  }
-
```

Comment:
@lcdservices looking at this hard-coded clause it is ordering by 3 fields: contact.sort_name, contact.id, and case.id. But I only see 2 fields added above.